### ISSUE TITLE
No hover, no intelligent suggestions

### ISSUE THREAD

**Issue by xNaCly** [fannheyward/coc-deno#262]
## TLDR

- no hover found
- no intelligent suggestions

## Problem

I have a singular typescript file (some details omitted for privacy, should still run):

```ts
async function lambda(): Promise<{ status: number; json: Record<string, any> }> {
    let body = { "hello": "world" };
    let req = await fetch("https://xnacly.me", {
        method: "POST",
        body: JSON.stringify(body),
    });
    return { status: req.status, json: {} };
}
console.log(await lambda());
```

I enable denols with `CocCommand deno.initializeWorkspace` (why is this required? Can't the plugin start upon noticing the `.ts` or `.js` extension?). After invoking `vim.fn.CocActionAsync('doHover')` i get the `[coc.nvim] hover not found` log, even when hovering over the function call and the function definition.

Am i doing something wrong or is the issue on your end?

## Details:
- deno version:
```
deno 2.0.0 (stable, release, x86_64-unknown-linux-gnu)
v8 12.9.202.13-rusty
typescript 5.6.2
```
- coc version: `0.0.82-9fd8578 2024-09-21 00:59:00 +0900`
- operating system: `Linux <hostname> 6.8.0-45-generic #45-Ubuntu SMP PREEMPT_DYNAMIC Fri Aug 30 12:02:04 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux`
- node version: `v22.9.0`
- neovim version: `NVIM v0.9.5`

## `CocInfo` Output:

```
2024-10-16T09:43:08.194 INFO (pid:24377) [services] - LanguageClient deno state change: stopped => starting
2024-10-16T09:43:08.197 WARN (pid:24377) [commands] - Command deno.initializeWorkspace already registered
2024-10-16T09:43:08.204 INFO (pid:24377) [language-client-index] - Language server "deno" started with 25888
2024-10-16T09:43:08.448 INFO (pid:24377) [services] - LanguageClient deno state change: starting => running
2024-10-16T09:43:08.457 INFO (pid:24377) [services] - service deno started
2024-10-16T09:44:12.624 INFO (pid:24377) [attach] - receive notification: runCommand []
2024-10-16T09:44:20.262 INFO (pid:24377) [attach] - receive notification: runCommand []
2024-10-16T09:44:23.833 INFO (pid:24377) [attach] - receive notification: doHover []
2024-10-16T09:44:24.264 INFO (pid:24377) [attach] - receive notification: doHover []
2024-10-16T09:44:25.656 INFO (pid:24377) [attach] - receive notification: doHover []
2024-10-16T09:44:29.875 INFO (pid:24377) [provider-formatManager] - Format by: coc-deno
2024-10-16T09:44:54.018 INFO (pid:24377) [attach] - Request action: fillDiagnostics [ 1 ]
2024-10-16T09:45:00.277 INFO (pid:24377) [provider-formatManager] - Format by: coc-deno
2024-10-16T09:45:14.982 INFO (pid:24377) [attach] - receive notification: doHover []
2024-10-16T09:49:54.374 INFO (pid:24377) [attach] - receive notification: showInfo []
```

## My coc and vim configuration

- https://github.com/xNaCly/dotfiles/tree/master/nvim

**Comment by xNaCly** [comment]
Well, this is akward, adding the `deno.enabled: true` to `coc-settings.json` fixed this, anyway - thanks for the great plugin, im loving deno :smile:
